### ISSUE TITLE
Can you add a https file server example?

### ISSUE THREAD

**Issue by ClemsonCoder** [wolkykim/libasyncd#35]
Hey.

I've been looking for a c/c++ library that can be used to make a https web server.

How do I combine the SSL and http server examples so that I can make an https server?

Best,

Rick

**Comment by wolkykim** [comment]
Yes I can. I need some time to work on that cause I am in Thailand spending
great time with the the ladies. Hold tight there. Will get back on this
soon. Thanks
Steve

**Comment by manjunathkokhle** [comment]
ad_server_t *server_tls = ad_server_new ();
   ad_server_set_option (server_tls, "server.port", *port*);
   ad_server_set_option (server_tls, "server.addr", *IP*);
//   ad_server_set_option (server_tls, "server.thread", "1"); if needed as separate thread
ad_server_set_ssl_ctx(server_tls, ad_server_ssl_ctx_create_simple(*cert file .pem* , *key file .pem* );
ad_server_register_hook(server_tls, ad_http_handler, NULL);
ad_server_start(server_tls);

this shd do the job..
